gtfs-realtime/spec/en/reference.md

```diff
@@ -1,6 +1,6 @@
 A GTFS Realtime feed lets transit agencies provide consumers with realtime information about disruptions to their service (stations closed, lines not operating, important delays, etc.) location of their vehicles, and expected arrival times.
 
-Version 2.0 of the feed specification is discussed and documented on this site.
+Version 2.0 of the feed specification is discussed and documented on this site. Valid versions are 2.0, 1.0.
```

Comment:
Add quotes ("2.0", "1.0") so consumers get a hint that they don't need to interpret these as anything other than strings.

Reply:
@TomGoBravo Good point!  I've added quotes to the text version numbers to make it clearer they are strings in https://github.com/google/transit/pull/130/commits/2ee02e7b9466fcb659bf43280516809c8f0679e3.